vendor/bat-native-ads/src/bat/ads/internal/client_state.cc

```diff
@@ -59,7 +59,8 @@ ClientState::ClientState(const ClientState& state) :
   search_activity(state.search_activity),
   search_url(state.search_url),
   shop_activity(state.shop_activity),
-  shop_url(state.shop_url) {}
+  shop_url(state.shop_url),
+  version_code (state.version_code) {}
```

Comment:
please change `version_code` to `app_version`

Reply:
NONCRITICAL
Covered by: brave/brave-browser#6235 (10)